applications/GeoMechanicsApplication/custom_utilities/transport_equation_utilities.hpp

```diff
@@ -79,6 +86,174 @@ class GeoTransportEquationUtilities
     {
         return -PORE_PRESSURE_SIGN_FACTOR * BiotModulusInverse * outer_prod(rNp, rNp) * IntegrationCoefficient;
     }
-    
+
+    static Matrix CalculateMassMatrixDiffOrder(const Geometry<Node>&          rGeom,
+                                               const Geometry<Node>::Pointer& rpPressureGeometry,
+                                               const GeometryData::IntegrationMethod IntegrationMethod,
+                                               const StressStatePolicy& rStressStatePolicy,
+                                               const std::vector<RetentionLaw::Pointer>& rRetentionLawVector,
+                                               const Properties&  rProp,
+                                               const ProcessInfo& rCurrentProcessInfo)
+    {
+        const SizeType dimension          = rGeom.WorkingSpaceDimension();
+        const SizeType number_U_nodes     = rGeom.PointsNumber();
+        const SizeType block_element_size = number_U_nodes * dimension;
+        const SizeType number_P_nodes     = rpPressureGeometry->PointsNumber();
+        const Geometry<Node>::IntegrationPointsArrayType& integration_points =
+            rGeom.IntegrationPoints(IntegrationMethod);
+
+        Matrix nu_container = rGeom.ShapeFunctionsValues(IntegrationMethod);
+        Matrix np_container = rpPressureGeometry->ShapeFunctionsValues(IntegrationMethod);
+        Vector pressure_vector =
+            GeoTransportEquationUtilities::GetSolutionVector(number_P_nodes, rGeom, WATER_PRESSURE);
+
+        // create general parameters of retention law
+        RetentionLaw::Parameters RetentionParameters(rProp, rCurrentProcessInfo);
```

Comment:
Should be snake_case
```suggestion
        RetentionLaw::Parameters retention_parameters(rProp, rCurrentProcessInfo);
```

Reply:
Thanks. changed.